README.md

```diff
@@ -80,40 +91,17 @@ You can then run `make build-charon-ml` to build the ML library, or even simply
 `make` to build the whole project (Rust and OCaml). Finally, you can run the
 tests with `make test`.
 
-Alternatively, you can use Nix and do `nix develop` (or use https://direnv.net/ and `direnv allow`)
-and all dependencies should be made available.
+Alternatively, you can use Nix and do `nix develop` and all dependencies should be made available.
 
 ## Documentation
 
 You can access the Rust documentation
 [online](https://aeneasverif.github.io/charon/charon_lib/index.html).
 
+As mentioned in [./docs/limitations.md], the project is currently poorly documented; we intend to
```

Comment:
```suggestion
```

Reply:
It's really badly documented, I'd rather keep the explicit paragraph so people know that we intend to do a much better job.

Reply:
It's not _that_ bad actually.